test_run_handler.go

```diff
@@ -37,19 +38,25 @@ func testRunHandler(w http.ResponseWriter, r *http.Request) {
 
 func handlePost(w http.ResponseWriter, r *http.Request) {
 	ctx := appengine.NewContext(r)
-	var err error
 
 	// Fetch pre-uploaded Token entity.
-	suppliedSecret := r.URL.Query().Get("secret")
+  var queryParams url.Values
```

Comment:
Nit: Strange indentation here?

Reply:
Fixed - I'm not sure which editor is doing that, I bounce between pycharm/gogland/sublime/atom